### PR TITLE
chore: enable custom port forward for minikube env

### PR DESCRIPTION
**Pull request by Alex-Izquierdo**
Allow to run for example `task minikube:fp:ui -- 9000` this is mostly for convenience and avoid potential collisions.

### CODE REVIEW

File: scripts/eda_kube.sh [hunk 1]
```diff
@@ -227,8 +228,8 @@ case ${CMD} in
   "build") build-all "${VERSION}" ;;
   "clean") clean-deployment "${VERSION}";;
   "deploy") deploy "${VERSION}" ;;
-  "port-forward-api") port-forward-api 8000 ;;
-  "port-forward-ui") port-forward-ui 8080 ;;
+  "port-forward-api") port-forward-api ${PORT} ;;
+  "port-forward-ui") port-forward-ui ${PORT} ;;
```
Comment (Dostonbek1) [inline scripts/eda_kube.sh:232]: Should this more specific? e.g. `API_PORT` & `UI_PORT`

**Review by Dostonbek1 — APPROVED** [review]

File: scripts/eda_kube.sh (hunk 1 above)
Comment (Alex-Izquierdo) [reply scripts/eda_kube.sh:232]: you will run only one command at a time.

Comment (Dostonbek1) [reply scripts/eda_kube.sh:232]: What if you wanted to forward ports for both? Anyway you will have to do it in a new separate terminal but in the end I think explicit > implicit.

Comment (Alex-Izquierdo) [reply scripts/eda_kube.sh:232]: the whole management of args is not a rebust implementation in this script I think, this is a small change to enable this, that's all. In fact I would like to remove it at some point in favor of ansible for various improvements for the minikube side.

**Review by hsong-rh — APPROVED** [review]

**Pull request merged**